lib/components/process-list/process-list-item.tsx

```diff
@@ -33,57 +33,55 @@ export const ProcessListItem = (props: ProcessListItemParameters): JSX.Element =
 
   const { error: tenureError, data: tenure } = useTenure(tenureId);
 
-  const address = tenure?.tenuredAsset?.fullAddress;
+  const address = !tenureError ? tenure?.tenuredAsset?.fullAddress : undefined;
 
   const createdAtStr = useMemo(() => formatDate(createdAt), [createdAt]);
 
   return (
     <div className="process">
-      {!tenureError && (
-        <>
-          <div className="process__item">
-            <div>
-              <Link
-                as={RouterLink}
-                to={`/person/${tenantId}`}
-                variant="link"
-                target="_blank"
-                className="lbh-heading-h4"
-              >
-                {fullName}
-              </Link>
-            </div>
-            <div style={{ marginTop: 0 }}>
-              <Link
-                as={RouterLink}
-                to={`/person/${tenantId}`}
-                variant="link"
-                target="_blank"
-              >
-                {address}
-              </Link>
-            </div>
-          </div>
-          <div className="process__item --processName">
+      <div className="process__item">
+        <div>
+          <Link
+            as={RouterLink}
+            to={`/person/${personId}`}
+            variant="link"
+            target="_blank"
+            className="lbh-heading-h4"
+          >
+            {fullName}
+          </Link>
+        </div>
+        {address && (
+          <div style={{ marginTop: 0 }}>
             <Link
               as={RouterLink}
-              to={`/processes/${processType}/${processId}`}
+              to={`/person/${personId}`}
```

Comment:
Shouldn't this refer to the property since it is displaying address?

Reply:
there is not such a requirement right now. In staging/prod it opens person detail page